bin/oneoff/back_populate_hoc_survey_prize_purpose

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env ruby
+
+# This script populates the purpose field in Pegasus.hoc_survey_prizes for claimed prizes with no purpose to "Hoc2014"
+
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+
+DB[:hoc_survey_prizes].exclude(claimant: nil).where(purpose: '').update(purpose: 'Hoc2014')
```

Comment:
How would you prefer it? Require a purpose for every prize and only allow claiming one if the purpose matches?

Note - currently they're not wired up to be claimed by anything. If we prefer this (or another approach), then when (if) we ramp up to use this table again on the next HOC we can make the change.